langchain/vectorstores/awadb.py

```diff
@@ -172,9 +216,14 @@ def similarity_search_with_relevance_scores(
         k: int = DEFAULT_TOPN,
         **kwargs: Any,
     ) -> List[Tuple[Document, float]]:
-        """Return docs and relevance scores, normalized on a scale from 0 to 1.
+        """Return docs the most similar to the text query.
```

Comment:
Was removing the portion about relevance scores intentional?

Reply:
Yes, the score has changed internally from L2 to InnerProduct, so the normalization can
not be needed again.

Reply:
Could you add a note about what the float it's returning now with each document represents?

Reply:
Good suggestion, I add a note about the changed relevance score.